### DISCUSSION TITLE
Feature Request: Custom Open Graph Data

### DISCUSSION THREAD

**Discussion by theigloo** [YOURLS/YOURLS#3037] [Ideas]
Hi 

Hope i am posting this in the right place :)

Services like rebrandly, linkly and others offer the option to ad custom open graph image and text for social media sharing

This would be great to have on YOURLS

Thanks

**Comment by ozh** [comment]
Is there anything that wouldn't work as a plugin ? Definitely sounds like plugin territory to me.

**Reply by theigloo** [reply]
I looked in plugiins couldnt find anything

**Comment by danielkuhnen** [comment]
Hope someone develops a plugin for this someday, really useful to have custom Open Graph image and text on the short link. I use this a lot on short.io
![open graph](https://github.com/YOURLS/YOURLS/assets/11381828/45ff8477-6058-408c-a04f-032bd66e865f)
